### ISSUE TITLE
Error on initial page load

### ISSUE THREAD

**Issue by pcragone** [brunofacca/zen-rails-base-app#6]
After following the instructions in the readme, I'm getting this error when I try to navigate to the page:

```
Started GET "/users/sign_in" for 172.16.1.106 at 2017-08-31 18:12:32 -0400
2017-08-31 18:12:32 -0400: Rack app error handling request { GET /users/sign_in }
#<NoMethodError: undefined method `*' for nil:NilClass>
Error reached top of thread-pool: undefined method `join' for nil:NilClass (NoMethodError)
```

I can't seem to figure out where it's coming from. Any ideas?

**Comment by brunofacca** [comment]
I was unable to reproduce your issue. The `*` and `join` may indicate a DB issue. Did you edit any migration files (`db/migrate/*`) before running `rails db:create db:migrate db:seed`?

**Comment by brunofacca** [comment]
Please type `User.all` within the Rails console and see if that also generates an error.

**Comment by pcragone** [comment]
Two things:
1: I discovered better_errors was hiding the entire stack trace; hence the highly cryptic message. I commented it out in the Gemfile and got a better stack trace, which leads to point 2
2: The error seemed to show a syntax error in devise, so I filed an issue with them as well. Take a look at this and let me know if it helps pinpoint things: https://github.com/plataformatec/devise/issues/4630

Long story short it sounds like this is not an issue with zen-rails-base-app, but rather an issue with somehow my environment is corrupted in some way. If you have an answer for that I'd certainly be appreciative in hearing it!  (If not, your rails dev environment vagrant setup works great!)

**Comment by pcragone** [comment]
Also, for completion's sake, typing `User.all` does not generate an error

**Comment by brunofacca** [comment]
The Ruby-2.5.0 thing you've mentioned in plataformatec/devise#4630 is odd. You could try [specifying a Ruby version in your  Gemfile](https://devcenter.heroku.com/articles/ruby-versions) but I'm not sure that will work.
